.gitignore

```diff
@@ -41,3 +41,18 @@ Thumbs.db
 # Filesystem storage
 data/*
 !data/.gitkeep
+
+# Registry Files
+packages/shared/src/lib/aggregates.ts
```

Comment:
The registry files should not be ignored by default. That should only happen when a developer runs the `prepare-contribution` command. Therefor, the command adds .gitignore files in the respecting folders so that registry files are ignored in contribution mode.

In normal mode, users will use Cody to generate an application and Cody needs to modify the registry files to register different components, messages, etc. 
And these modified registry files need to be committed along with the rest of the generated code.

Only in contribution mode, we don't want to commit changes in the registry files, because the skeleton should stay clean.

Reply:
ah okay, makes sense!